scripts/standalone.mk

```diff
@@ -37,7 +37,16 @@ ifeq ($(RISCV_CMODEL),)
 RISCV_CMODEL = medany
 endif
 
-ifeq ($(LINK_TARGET),)
+ifeq ($(PROGRAM),dhrystone)
+LINK_TARGET = ramrodata
+# Ground rules
+RISCV_XCFLAGS += -mexplicit-relocs -fno-inline
+# With inline and without lto
+#RISCV_XCFLAGS += -mexplicit-relocs -finline
+# Best Score
+#RISCV_XCFLAGS += -finline -flto -fwhole-program
+RISCV_XCFLAGS += -DDHRY_ITERS=$(TARGET_DHRY_ITERS)
+else
```

Comment:
Should be 
```
else ifeq ($(LINK_TARGET),)
```
to preserve the original behavior, otherwise users won't be able to select a different link target.